projects/core/bindaas-core-impl/src/main/java/edu/emory/cci/bindaas/core/jwt/UserInfo.java

```diff
@@ -45,6 +45,10 @@
 	@Expose
 	private Boolean emailVerified;
 
+	@SerializedName("https://bindaas.com/role")
```

Comment:
Can you get this name from a constant somewhere?

Reply:
This a POJO for the response we get from auth0. The response is in JSON format and I have defined the other keys in the same way. Since this format will not change, do we need a constant. If yes, should I change all other values as well?

Reply:
I think it is best to keep it in the constant file that you are creating for easier access and better documentation.